### DISCUSSION TITLE
Diagram sort order appears to be using string sorting?

### DISCUSSION THREAD

**Discussion by AndyTeas** [structurizr/lite#56] [Q&A] [answered]
### Description

With a set of DSL files with more than 10 diagrams, the 10th diagram occupies position 2 in the left hand diagram list
![Screenshot 2023-04-07 145357](https://user-images.githubusercontent.com/5708057/230621082-932e56dc-2152-43e3-be37-1005b2a02fe6.png)

### Steps to reproduce

1, create a model with more than 9 diagrams.

### Configuration

Structurizr Lite
- build: 3047 (2023-04-06T07:33:20Z
- structurizr-java: v1.24.1
- structurizr-dsl: v1.30.1
- structurizr-import: v1.4.1

### Severity

Minor

### Priority

Medium

### Resolution

I have no budget, please fix this for free

**Comment by simonbrowndotje** [comment]
Can't reproduce; converting to discussion...

**Comment by simonbrowndotje** [comment, marked as answer]
Without a specific DSL example to reproduce what you're seeing, it's hard to say what's happening I'm afraid. I suspect what you're seeing is correct though; see 
 https://structurizr.com/help/diagrams#sorting for details of how diagrams are sorted.

**Reply by AndyTeas** [reply]
Thanks!  Had missed that, add:
`
        properties {
            "structurizr.sort" "created"
        }
`
has put things in the order I was expecting.
